docs/docker_instructions.md

```diff
@@ -6,7 +6,7 @@ The Docker CLI is a command-line tool with a whole library of commands for inter
 
 
 ### Part A - Running Containers
-In this section you will try running a basic container called whalesay. Refer to the [part A readings](#part-a-readings) if you need a refresher on the content taught in the training.
+In this section you will try running a basic container called whalesay. Refer to part A below if you need a refresher on the content taught in the training.
```

Comment:
What's the motivation for this change?  We're in "Part A" already, and the other "Part A" is a long way down the page (past another "Part A"), so the link is helpful.

Reply:
The immediate motivation is something about this method of linking in a md (vs rst) document is killing the build hard.  Presumably some dependency has changed it's behaviour. 

Exact error message is 
```
/root/project/docs/docker_instructions.md:9: WARNING: 'myst' cross-reference target not found: 'part-a-readings' [myst.xref_missing]
/root/project/docs/docker_instructions.md:39: WARNING: 'myst' cross-reference target not found: 'part-b-readings' [myst.xref_missing]
```

I'll try to describe it more.

Reply:
Ok, found the breaking change

Reply:
yay